tests/e2e/journeys/upgrade-free-tier.spec.ts

```diff
@@ -7,7 +7,7 @@ test('upgrade - free tier', async ({ page }) => {
     await registerUserStep(page);
     await createFreeProject(page);
     await test.step('upgrade project', async () => {
-        await page.getByRole('link', { name: 'upgrade' }).click();
+        await page.getByRole('link', { name: 'Upgrade', exact: true }).click();
```

Comment:
how come this needs to be exact now?

Reply:
If there is any 'update' text in an anchor link with higher z-index than the main content, e2e seems to fail. Prior commits x tests can be seen where this happens. Assuming the e2e calls the Update in the header, this should fix it.